controllers/webspherelibertyapplication_controller.go

```diff
@@ -140,6 +140,39 @@ func (r *ReconcileWebSphereLiberty) Reconcile(ctx context.Context, request ctrl.
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
+
```

Comment:
We should minimize the code in the controllers and share as much as possible between the 3 runtime operators. It'll be good to refactor this into a utility method (that returns an error if there is conflict, nil otherwise) and add to RCO.